### PR TITLE
Added changelog for 1.7.0-alpha03

### PR DESCRIPTION
**Pull request by mazunin-v-jb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+- [Skiko version was updated to 0.8.11 (It is based on skia m126 release)](https://github.com/JetBrains/compose-multiplatform-core/pull/1486)
+
+## Features
+### Multiple Platforms
+- [Commonized `material3-window-size-class` module](https://github.com/JetBrains/compose-multiplatform-core/pull/1466)
+- [Commonized `material3-common` module](https://github.com/JetBrains/compose-multiplatform-core/pull/1465)
+- [Commonized `:compose:material3:adaptive:adaptive`, `:compose:material3:adaptive:adaptive-layout`, `:compose:material3:adaptive:adaptive-navigation` and `:window:window-core` modules](https://github.com/JetBrains/compose-multiplatform-core/pull/1468)
+
+### iOS
+- [Opt-out of new touches strategy on iOS](https://github.com/JetBrains/compose-multiplatform-core/pull/1460)
```
Comment (MatkovIvan) [inline CHANGELOG.md:16]: It's not valid

@elijah-semyonov please remove it from description (not just apply strikethrough formatting)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+## Dependencies
+- Gradle Plugin `org.jetbrains.compose`, version `1.7.0-alpha02`. Based on Jetpack Compose libraries:
+  - [Runtime 1.7.0-beta06](https://developer.android.com/jetpack/androidx/releases/compose-runtime#1.7.0-beta06)
+  - [UI 1.7.0-beta06](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.7.0-beta06)
+  - [Foundation 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.7.0-beta06)
+  - [Material 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-material#1.7.0-beta06)
+  - [Material3 1.3.0-beta03](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.3.0-beta05)
+
+- Lifecycle libraries `org.jetbrains.androidx.lifecycle:lifecycle-*:2.8.0`. Based on [Jetpack Lifecycle 2.8.0](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.0)
+- Navigation libraries `org.jetbrains.androidx.navigation:navigation-*:2.8.0-alpha08`. Based on [Jetpack Navigation 2.8.0-beta05](https://developer.android.com/jetpack/androidx/releases/navigation#2.8.0-beta05)
```
Comment (MatkovIvan) [inline CHANGELOG.md:41]: alpha08 was already release last time

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+### Gradle Plugin
+- [_(prerelease fix)_ Fix broken configuration cache due Android Studio + AGP issues. Now Android Studio previews require latest AGP versions (8.5.2, 8.6.0-rc01, 8.7.0-alpha04): https://issuetracker.google.com/issues/348208777](https://github.com/JetBrains/compose-multiplatform/pull/5118)
+
+## Dependencies
+- Gradle Plugin `org.jetbrains.compose`, version `1.7.0-alpha02`. Based on Jetpack Compose libraries:
+  - [Runtime 1.7.0-beta06](https://developer.android.com/jetpack/androidx/releases/compose-runtime#1.7.0-beta06)
+  - [UI 1.7.0-beta06](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.7.0-beta06)
+  - [Foundation 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.7.0-beta06)
+  - [Material 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-material#1.7.0-beta06)
+  - [Material3 1.3.0-beta03](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.3.0-beta05)
```
Comment (MatkovIvan) [inline CHANGELOG.md:38]: text and link version mismatch

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,47 @@
...
+
+### Gradle Plugin
+- [_(prerelease fix)_ Fix broken configuration cache due Android Studio + AGP issues. Now Android Studio previews require latest AGP versions (8.5.2, 8.6.0-rc01, 8.7.0-alpha04): https://issuetracker.google.com/issues/348208777](https://github.com/JetBrains/compose-multiplatform/pull/5118)
+
+## Dependencies
+- Gradle Plugin `org.jetbrains.compose`, version `1.7.0-alpha02`. Based on Jetpack Compose libraries:
+  - [Runtime 1.7.0-beta06](https://developer.android.com/jetpack/androidx/releases/compose-runtime#1.7.0-beta06)
+  - [UI 1.7.0-beta06](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.7.0-beta06)
+  - [Foundation 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.7.0-beta06)
+  - [Material 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-material#1.7.0-beta06)
```
Comment (MatkovIvan) [inline CHANGELOG.md:37]: text and link version mismatch

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,47 @@
...
+- [_(prerelease fix)_  Fix the bug where only the changed touches were sent Compose, while all tracked touches were expected](https://github.com/JetBrains/compose-multiplatform-core/pull/1477)
+
+### Gradle Plugin
+- [_(prerelease fix)_ Fix broken configuration cache due Android Studio + AGP issues. Now Android Studio previews require latest AGP versions (8.5.2, 8.6.0-rc01, 8.7.0-alpha04): https://issuetracker.google.com/issues/348208777](https://github.com/JetBrains/compose-multiplatform/pull/5118)
+
+## Dependencies
+- Gradle Plugin `org.jetbrains.compose`, version `1.7.0-alpha02`. Based on Jetpack Compose libraries:
+  - [Runtime 1.7.0-beta06](https://developer.android.com/jetpack/androidx/releases/compose-runtime#1.7.0-beta06)
+  - [UI 1.7.0-beta06](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.7.0-beta06)
+  - [Foundation 1.7.0-beta05](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.7.0-beta06)
```
Comment (MatkovIvan) [inline CHANGELOG.md:36]: text and link version mismatch

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,47 @@
...
+
+_Changes since 1.7.0-alpha02_
+
+## Highlights
+### Multiple Platforms
+- [Skiko version was updated to 0.8.11 (It is based on skia m126 release)](https://github.com/JetBrains/compose-multiplatform-core/pull/1486)
+
+## Features
+### Multiple Platforms
+- [Commonized `material3-window-size-class` module](https://github.com/JetBrains/compose-multiplatform-core/pull/1466)
```
Comment (igordmn) [inline CHANGELOG.md:11]:
```suggestion
- [Commonized `org.jetbrains.compose.material3:material3-window-size-
class` module](https://github.com/JetBrains/compose-multiplatform-core/pull/1466)
```

**Review by igordmn — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,50 @@
+# 1.7.0-alpha03 (August 2024)
+
+_Changes since 1.7.0-alpha02_
+
+## Features
+### Multiple Platforms
+- [Skia is updated to m126](https://github.com/JetBrains/compose-multiplatform-core/pull/1486)
+- [Commonized `org.jetbrains.compose.material3:material3-window-size-class` module](https://github.com/JetBrains/compose-multiplatform-core/pull/1466)
+- [Commonized `org.jetbrains.compose.material3.adaptive:adaptive-*` modules](https://github.com/JetBrains/compose-multiplatform-core/pull/1468)
```
Comment (zamulla) [inline CHANGELOG.md:9]:
```suggestion
- [Commonized `org.jetbrains.compose.material3.adaptive:adaptive*` modules](https://github.com/JetBrains/compose-multiplatform-core/pull/1468)
```

Should the dash be there? In [the PR](https://github.com/JetBrains/compose-multiplatform-core/pull/1468/files#diff-6fe33dbc3032d362b36542545eb0a488f590fb05af7f9cb10e1865313c63e15e) the `material3.adaptive:adaptive` module seems to also be included.

Comment (mazunin-v-jb) [reply CHANGELOG.md:9]: thx, I've mistaken here

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,50 @@
...
+- [_(prerelease fix)_ Fix input methods position on the screen and `NullPointerException: Cannot read field`](https://github.com/JetBrains/compose-multiplatform-core/pull/1491)
+
+### iOS
+- [_(prerelease fix)_  Fix the bug where only the changed touches were sent Compose, while all tracked touches were expected](https://github.com/JetBrains/compose-multiplatform-core/pull/1477)
+
+### Gradle Plugin
+- [_(prerelease fix)_ Fix broken configuration cache due Android Studio + AGP issues. Now Android Studio previews require latest AGP versions (8.5.2, 8.6.0-rc01, 8.7.0-alpha04): https://issuetracker.google.com/issues/348208777](https://github.com/JetBrains/compose-multiplatform/pull/5118)
+
+## Dependencies
+- Gradle Plugin `org.jetbrains.compose`, version `1.7.0-alpha02`. Based on Jetpack Compose libraries:
```
Comment (MatkovIvan) [inline CHANGELOG.md:26]: `1.7.0-alpha02` -> `1.7.0-alpha03`

**Pull request merged**